flipdisc/framework/app.py

```diff
@@ -326,7 +368,7 @@ def setup_output(self, cfg="output_stream", bind=None):
             self._out_topic,
         )
 
-    async def send_output(self, result, topic=None) -> int:
+    async def send_output(self, result: numpy.ndarray, topic: str=None) -> int:
```

Comment:
Not 100% sure on style, and it's probably fine, but I think these should be explicitly marked optional. It may be that MonkeyType deliberately disobeys [PEP-484](https://peps.python.org/pep-0484/#union-types) here.

```suggestion
    async def send_output(self, result: numpy.ndarray, topic: Optional[str]=None) -> int:
```